gui/templates/system/debug.html

```diff
@@ -10,7 +10,11 @@
     </script>
 <table class="form">
     {% csrf_token %}
+    {% if failover_down %}
+        <tr><td style="color: red">Standby node is not available. The debug is going to contain data from the standby node only. </td></tr>
```

Comment:
As @vaibhav-rbs stated this should be: "... The debug is going to contain data from the active node only"